treeAnalysis/jetresolutionhistos.cxx

```diff
@@ -195,6 +199,8 @@ void jetresolutionhistos(std::tuple<std::shared_ptr<fastjet::ClusterSequenceArea
         if(verbosityJRH>1) std::cout << "rec jet " << i << " (" << std::get<1>(recjets)[i].E() << "E)  matched with true jet " << j << " (" << std::get<1>(truejets)[j].E() << "E) with dR = " << deltaRTrueRec << endl;
         h_jetscale_E[select][et]->Fill(std::get<1>(truejets)[j].E(),(std::get<1>(recjets)[i].E()-std::get<1>(truejets)[j].E())/std::get<1>(truejets)[j].E());
         h_jetscale_pT[select][et]->Fill(std::get<1>(truejets)[j].pt(),(std::get<1>(recjets)[i].pt()-std::get<1>(truejets)[j].pt())/std::get<1>(truejets)[j].pt());
+        h_jetscale_eta[select][et]->Fill(std::get<1>(truejets)[j].eta(),(std::get<1>(recjets)[i].eta()-std::get<1>(truejets)[j].eta()));
+        h_jetscale_phi[select][et]->Fill(std::get<1>(truejets)[j].phi(),(std::get<1>(recjets)[i].phi()-std::get<1>(truejets)[j].phi()));
```

Comment:
phi is periodic, so you should use `delta_phi_to` here like it's used a few lines below.

Reply:
Good catch, thanks!